pkg/resourcehandlers/git/gitGinko_resource_handler_test.go

```diff
@@ -0,0 +1,734 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Can you transfer changes from https://github.com/gardener/docforge/blob/54cd58579436fd8fe2c558e4d4df5d93bab4c5ff/pkg/resourcehandlers/github/githubGinko_resource_handler_test.go to this PR